### DISCUSSION TITLE
Why not replace the suffix of configuration .data  with .json?

### DISCUSSION THREAD

**Discussion by ByteFlyCoding** [bfenetworks/bfe#1091] [General]
EN: As we known, the formate of bfe configuration file is json format. Why not replace the suffix of configuration .data  with .json?
If the file suffix is .json, it is useful that Ide or edit can format it, support intellisense and remind editor problem.

CH: 我们知道，bfe 配置文件是json格式的，为什么不把配置文件的文件后缀从.data换成.json？如果文件后缀是.json的话，这是非常有用的，Ide和编辑器可以为.json格式结尾的文件提供格式化、智能感知与提示及提醒编辑者错误等功能。

<img width="325" alt="image" src="https://user-images.githubusercontent.com/81181757/204809870-82cd0eac-db2e-4bb1-9d99-a9bd0421c6e0.png">

**Comment by iyangsj** [comment]
bfe这类配置并不依赖人工编辑，而是由平台管理，并不需要强制使用json后缀

@ByteFlyCoding 你可以按需自由定制文件路径及后缀名称，使用.json作为后缀（bfe仅要求是合法json格式即可）
